src/components/course-review-card.js

```diff
@@ -5,15 +5,16 @@ import { Card } from 'semantic-ui-react';
 import { Rating } from 'semantic-ui-react';
 
 const CourseReviewCard = (props) => {
-  const { code } = props;
+  const { code, desc } = props;
```

Comment:
a course description would actually be a bigger chunk, so save this variable name for the actual description. i reckon `name` for course name would be better